dashboard/setup.md

```diff
@@ -0,0 +1,149 @@
+# Dashboard
+
+The Catena admin dashboard is a [React](https://react.dev/reference/react) application that is intended to run at the same address as the `catena-tools-core` backend.
```

Comment:
> intended

Let's clarify this as it's often done this way but it not required

(I know this came from the README)